### PR TITLE
Add installation script

### PR DESCRIPTION
**Pull request by Schniz**
A work in progress.
Not sure if I'd like to cover it with a "feature" test too

Can be a great workaround until #5 will work out

### CODE REVIEW

**Comment by Schniz** [comment]
@elliottsj, can you please take a look at the fish section? 😄

File: .ci/install.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    echo 'export PATH=$HOME/.fnm:$PATH' >> $HOME/.zshrc
+    echo 'eval `fnm env`' >> $HOME/.zshrc
+
+  elif [ "$CURRENT_SHELL" == "fish" ]; then
+    echo "Installing for Fish. Appending the following to $HOME/.config/fish/config.fish:"
+    echo 'set PATH $HOME/.fnm $PATH'
+    echo 'eval (fnm env)'
+
+    echo 'set PATH $HOME/.fnm $PATH' >> $HOME/.config/fish/config.fish
+    echo 'eval (fnm env)' >> $HOME/.config/fish/config.fish
```
Comment (elliottsj) [inline .ci/install.sh]:
```suggestion
    echo 'eval (fnm env --fish)' >> $HOME/.config/fish/config.fish
```

File: .ci/install.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    echo 'export PATH=$HOME/.fnm:$PATH'
+    echo 'eval `fnm env`'
+
+    echo 'export PATH=$HOME/.fnm:$PATH' >> $HOME/.zshrc
+    echo 'eval `fnm env`' >> $HOME/.zshrc
+
+  elif [ "$CURRENT_SHELL" == "fish" ]; then
+    echo "Installing for Fish. Appending the following to $HOME/.config/fish/config.fish:"
+    echo 'set PATH $HOME/.fnm $PATH'
+    echo 'eval (fnm env)'
```
Comment (elliottsj) [inline .ci/install.sh]:
```suggestion
    echo 'eval (fnm env --fish)'
```

**Comment by Schniz** [comment]
Cool! thanks! 😄

**Comment by Schniz** [comment]
Okay. I thought Fish doesn't provide the `$SHELL` env var, but it seems that it does. `chsh -s $(which fish)` fixed that, so I'm merging.

**Pull request merged**